### PR TITLE
Completed Works Section & About Myself Section

### PR DESCRIPTION
**Pull request by CVILLA09**
This pull request includes the completed mobile version of the portfolio section. I have implemented the "Works" section and the "About myself" section as per the project requirements. The content is aligned using flexbox and CSS Grid combined. I have also ensured that the website is responsive and follows the specified breakpoints. The design has been closely matched to the Figma template provided. Please review and confirm in order to merge the changes into the main branch.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,18 +23,198 @@
...
+        </div>
+        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="project-tags">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">CSS</a></li>
+          <li><a href="#">JavaScript</a></li> 
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+    </article>
```
Comment (abdulhamiid) [inline index.html:61]: According to the[ project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), kindly use `Flexbox` to lay out an individual card. 😊🛠

* This gives us a lot of options to align all our elements, plus, it's responsive.

File: index.html [hunk 2]
```diff
@@ -21,18 +23,198 @@
...
     </section>
+<!-- Projects Section -->
+<section class="project-section" aria-label="Projects">
+    <!-- 1st project card -->
+    <article class="project-card"> 
+      <div class="project-img"> 
+        <img src="assets/Snapshoot-1.svg" alt="Screenshot of Project 1">
+      </div>
+      <div class="project-content"> 
+        <h2>Tonic</h2>
```
Comment (abdulhamiid) [inline index.html:47]: To ensure that we stick to stick to the design as much as possible, let's use the same color for the title as Figma 👍😎

Your design
image

Figma
image

File: index.html [hunk 3]
```diff
@@ -21,18 +23,198 @@
...
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">CSS</a></li>
+          <li><a href="#">JavaScript</a></li>
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
+      </div>
+    </article>
+  </section>
+   <!-- ABOUT SECTION -->
+   <section id="about-section" class="about-section">
```
Comment (abdulhamiid) [inline index.html]: Please use `Flexbox` to align the content of the "About myself" section as stated in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), 👍🛠

File: index.html [hunk 4]
```diff
@@ -21,18 +23,198 @@
...
+            <ul class="skills-container active">
+                <li>
+                    <h2 class="skills-title">Skills</h2>
+                </li>
+                <li>
+                    <figcaption class="skills-image-container">
+                        <img src="assets/arrow.svg" alt="arrow-vector" class="skills-image">
+                    </figcaption>
+                </li>
+            </ul>
```
Comment (abdulhamiid) [inline index.html]: According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), please use `ul` and `li` for categories in "About myself" section. i.e. `Languages`, `Frameworks`, and `Skills` should be list items.


Example
```HTML
<ul>
  <li>
    <h3>Languages</h3>
    . . . 
    <ul>
       <li>JavaScript</li>
       <li> . . .  </li>
       <li> . . .  </li>
    </ul>
  </li>
  <li>
    <h3>Frameworks</h3>
     . . . 
  </li>
  <li> 
   <h3>Skills</h3>
    . . . 
  </li>
</ul>
```

Kindly note that the categories are wrapped in a single `ul` tag 👀🤗

File: style.css [hunk 5]
```diff
@@ -109,3 +118,231 @@ ul {
...
+/* ABOUT SECTION */
+.about-section {
+  margin-top: 0;
+}
+
+.about-container {
+  background-color: #fff;
+  padding: 114px 0;
+  border-radius: 0 15% 0 0;
+}
```
Comment (abdulhamiid) [inline style.css]: Your design
image

Figma
image

I would suggest that we use pixels to enable a smooth border-radius similar to the design in Figma. 🤓

example
```CSS
border-radius: 0 80px 0 0;
```

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @CVILLA09 👋, 
Alt Text
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Understand how to parse a Figma design to create a UI.
- ✔️ Use Flexbox to place elements in the page.
- ✔️ Use CSS Grid to place elements in the page.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -21,18 +23,203 @@
...
+              </li>
+              <li class="skills-item">
+                  <div class="language-item">
+                      <figcaption class="language-image-container">
+                          <img src="assets/css.png" alt="css" class="language-image">
+                      </figcaption>
+                      <h2 class="skills-item-title">CSS</h2>
+                  </div>
+              </li>
+          </ul>
```
Comment (frempongdev) [inline index.html]: - [ ] As pointed out by your earlier reviewer, please make sure that all the technology categories (`"Languages"`, `"Frameworks"` and `"Skills"`) are in a `<ul>`, each of them in a `<li>` 🦭 

Kindly follow the guidelines below provided by the earlier reviewer 🎯 

image

From the above, each technology section gets its own `<li>` and are all found inside one `<ul>`
The technologies can also make use of sub `<ul>'s` and `<li>'s` to make their own sub-lists 🪜 

Hence
**The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`**

File: style.css [hunk 7]
```diff
@@ -109,3 +118,236 @@ ul {
 .social-icons li {
   padding-right: 18px;
 }
+
+/* Project Section */
+.project-section {
+  display: flex;
```
Comment (frempongdev) [inline style.css]: [OPTIONAL]
- [ ] Good job using `display: flex;` in the main container for the project cards, however the project requirement is to use a `display: grid` for the main container of the work section. ↔️ 👍🏿

**Review by frempongdev — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @CVILLA09   👋🏼 
I'm Frempong.

You've done a great job implementing most of the requirements for this project 🚀, but there are a few changes I recommend be made. You're almost done, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 


 ## Required Changes ♻️
_Kindly Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 8]
```diff
@@ -21,18 +23,199 @@
...
+         </li>
+         <li>
+         <div class="title-container">
+           <h3 class="skills-title">Skills</h3>
+           <ul class="skills-container active">
+             <li>
+               <figcaption class="skills-image-container">
+                 <img src="assets/arrow.svg" alt="arrow-vector" class="arrow-image">
+               </figcaption>
+               </div>
```
Comment (Efiamotu-1) [inline index.html]: [OPTIONAL]
- [ ] The `div` in this highlighted line is not properly placed as an enclosing closing tag. kindly fix this for better html structure before merging :+1: :sparkles:

**Review by Efiamotu-1 — APPROVED** [review]
Hi @CVILLA09 . 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PRItle & summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**